docs/getstarted/discovery-hub.mdx

```diff
@@ -191,7 +191,7 @@ Click on the **+ Add schedule button** to schedule the scan.
 
 #### Additional Details
 
-This information is accurate as of release v18.12.00, with more usability enhancements on their way.
+Device42 is continuously works to add new usability enhancements.
```

Comment:
This provides little value, remove sentence and ensure the rest of the page makes sense.

Reply:
Removed